src/sentry/snuba/trace.py

```diff
@@ -368,8 +367,10 @@ def _serialize_columnar_uptime_item(
     project_id = row_dict["sentry.project_id"].val_int
     project_slug = project_slugs[project_id]
 
+    item_id_str = row_dict["sentry.item_id"].val_str
+
     span = {
-        "event_id": guid,
```

Comment:
Any value in still returning the `check_guid` or something I wonder?

Reply:
It should be included in the attributes